arches/app/templates/javascript.htm

```diff
@@ -380,6 +380,9 @@
             },{% endfor %}{% endautoescape %}],
             userEmail: '{{ user.email }}',
             celeryRunning: '{{celery_running}}',
+            export_html_templates: [{% autoescape off %}{% for template in export_html_templates %}
```

Comment:
Could the key `export_html_template` be changed to `exportHtmlTemplate` just to keep with camel case on the JavaScript side? Line 22 in `views/components/search/search-export.js` would need to be updated to reflect the change.

Reply:
Was in PEP8 mode! Corrected.